.travis.yml

```diff
@@ -11,9 +11,14 @@ matrix:
         - python: "3.7"
           dist: xenial
           sudo: true
+        - python: "3.8"
+          dist: xenial
+          sudo: true
```

Comment:
dist: xenial is default nowadays; sudo: true as well.  All of these can be simplified to

```
python:
    - 2.7
    - 3.5
    - 3.6
    - 3.7
    - 3.8
```

(https://pypi.org/project/check-python-versions/ can do that for you!)

Reply:
Ah, wonderful! I did not know that one yet. Works like a charm!